src/UKHO.ERPFacade.Common/IO/AzureBlobEventWriter.cs

```diff
@@ -14,6 +17,7 @@ public class AzureBlobEventWriter : IAzureBlobEventWriter
     {
         private readonly ILogger<AzureBlobEventWriter> _logger;
         private readonly IOptions<AzureStorageConfiguration> _azureStorageConfig;
+        private Task<bool> isExists;
```

Comment:
Not sure what this is for? Please remove if not used - if it is, please rename to indicate what it does, and use _ prefix convention

Reply:
done.